extensions/godslayerakp/ws.js

```diff
@@ -0,0 +1,283 @@
+// Name: WebSocket
+// Description: an extension for connecting to websocket servers like discord and cloudlink.
+// By: RedMan13 <https://scratch.mit.edu/users/RedMan13/>
+
+(function (Scratch) {
+  "use strict";
+
+  if (!Scratch.extensions.unsandboxed) {
+    throw new Error("can not load outside unsandboxed mode");
+  }
+
+  /* ------- BLOCKS -------- */
+  const { BlockType, Cast, ArgumentType } = Scratch;
+
+  class WebSocketExtension {
+    /**
+     * no need to install runtime as it comes with Scratch var
+     */
+    constructor() {
+      this.instances = {};
+    }
+    getInfo() {
+      return {
+        id: "websocket",
+        name: "WebSocket",
+        color1: "#307eff",
+        color2: "#2c5eb0",
+        blocks: [
+          {
+            opcode: "newInstance",
+            blockType: BlockType.COMMAND,
+            arguments: {
+              URL: {
+                type: ArgumentType.STRING,
+                defaultValue: "wss://example.com",
```

Comment:
also, if you happen to know of a public WebSocket echo server of some sort, consider putting that here instead

Reply:
> also, if you happen to know of a public WebSocket echo server of some sort, consider putting that here instead

i dont but if you have any suggestions i would be fine with using them

Reply:
for now i think ima just make my own with replit for now :/